### PR TITLE
docs: :memo: decision post on why to use the Frictionless Data standard

### LINKED ISSUE #96: Write a why post on using frictionless standard

### PR DESCRIPTION
**Pull request by lwjohnst86**
## Description

This decision post describes why we should use the Frictionless Data standard for structuring and extracting metadata from data files.

Closes #96 

## Reviewer Focus

This PR needs a bit of an in-depth review. 

## Checklist

- [x] Ran spell-check
- [x] Formatted Markdown
- [x] Rendered website locally

### CODE REVIEW

File: why-frictionless-data/index.qmd [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+-   The overall design and architecture of the standard is easy to
+    understand and well designed.
+-   Interacts with multiple programming languages and services,
+    including Python and GitHub.
+-   Has clearly described principles and values, which make it easy to
+    understand the intent of the standard.
+-   Everything is stored in a JSON file, which makes it
+    machine-readable.
+-   The Python package provides functionality to extract the metadata of
+    data files and to create
```
Comment (martonvago) [inline why-frictionless-data/index.qmd:110]: Looks like the end of the sentence is missing

File: why-frictionless-data/index.qmd [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+REASONS."
+:::
+
+While there are no real alternatives to Frictionless Data that fit our
+requirements, there are many strong reasons to use it as is described
+above. For instance, the principles and values align closely with ours
+and the organization who owns and is responsible for development of this
+standard, which is the Open Knowledge Foundation and is a non-profit
+organization, does work in other areas that also align with our values
+and principles.
```
Comment (martonvago) [inline why-frictionless-data/index.qmd:138]:
```suggestion
and this standard is owned and developed by the non-profit Open 
Knowledge Foundation, who do work in other areas that also align 
with our values and principles.
```
Something like this maybe?

File: why-frictionless-data/index.qmd [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    OpenDataDiscovery.
+
+The only real open metadata standard that seems to exist is the
+[Frictionless Data](https://frictionlessdata.io/) standard, which
+surprisingly wasn't listed in the resources above (likely for the
+opposite reasons we didn't consider the tools they list).
+
+### Frictionless Data
+
+[Frictionless Data](https://frictionlessdata.io/) and it's companion
```
Comment (martonvago) [inline why-frictionless-data/index.qmd:90]:
```suggestion
[Frictionless Data](https://frictionlessdata.io/) and its companion
```

**Review by martonvago — CHANGES_REQUESTED** [review]
Looks like this is a great fit! 😁

File: why-frictionless-data/index.qmd [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+    intended as a platform for hosting and sharing (internal) data.
+    There is a command-line interface, but there is no easy
+    documentation on how to use it, nor is it actively developed.
+-   [OpenMetadata](https://docs.open-metadata.org/v1.4.x): Similar to
+    OpenDataDiscovery.
+
+The only real open metadata standard that seems to exist is the
+[Frictionless Data](https://frictionlessdata.io/) standard, which
+surprisingly wasn't listed in the resources above (likely for the
+opposite reasons we didn't consider the tools they list).
```
Comment (lwjohnst86) [inline why-frictionless-data/index.qmd:86]:
```suggestion
surprisingly wasn't listed in the resources above. This is likely for the
opposite reasons we didn't consider the tools they list.
```

File: why-frictionless-data/index.qmd [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+-   The overall design and architecture of the standard is easy to
+    understand and well designed.
+-   Interacts with multiple programming languages and services,
+    including Python and GitHub.
+-   Has clearly described principles and values, which make it easy to
+    understand the intent of the standard.
+-   Everything is stored in a JSON file, which makes it
+    machine-readable.
+-   The Python package provides functionality to extract the metadata of
+    data files and to create
```
Comment (lwjohnst86) [inline why-frictionless-data/index.qmd:110]:
```suggestion
    data files and to validate the input data against the metadata.
```

**Pull request merged**